build.gradle

```diff
@@ -92,12 +92,14 @@ publishing {
             def releasesRepoUrl = "https://s01.oss.sonatype.org/service/local/staging/deploy/maven2/"
             def snapshotsRepoUrl = "https://s01.oss.sonatype.org/content/repositories/snapshots/"
             url = isReleaseVersion ? releasesRepoUrl : snapshotsRepoUrl
+            println("url $url")
+
             credentials {
-                username = System.getenv("MAVEN_USERNAME")
-                password = System.getenv("MAVEN_PASSWORD")
+                username = ossrhUsername
+                password = ossrhPassword
             }
         }
-        // This currently only works for released versions, as github has problems with overwriting snapshots
+       /* // This currently only works for released versions, as github has problems with overwriting snapshots
```

Comment:
Can we try to craft GitHub releases? (Benefit?)

Reply:
I don't know where this comment came from